practice-app/website/test/test_forum.py

```diff
@@ -3,9 +3,12 @@
 sys.path.append('./website') #to run the test without import errors, one should call from one directory above this line's directory
 sys.path.append('.')
 
```

Comment:
Unittest cases look good, I have just a suggestion to make unit tests for GET requests, too.

Reply:
I skipped that since it required changes in the api/discussion_forum.py file.